docs/reference/security/authentication/jwt-realm.asciidoc

```diff
@@ -181,7 +183,19 @@ Instructs the realm to treat and validate incoming JWTs as access tokens (`acces
 Specifies a list of JWT subjects that the realm will allow.
 These values are typically URLs, UUIDs, or other case-sensitive string values.
 
-NOTE: This setting is mandatory for when `token_type` is `access_token`.
+`allowed_subject_patterns`::
+Analogous to `allowed_subjects` but it accepts a list of <<regexp-syntax,Lucene regexp>>
+and wildcards for the allowed JWT subjects. Wildcards use the `*` and `?` special
+characters (which are escaped by `\`) to mean "any string" and "any single character"
```

Comment:
optional: Would be nice to include a small example (similarly like you did for Lucene regex) that shows usage of both wildcards including escaping.

Reply:
Pushed [cc62b3b](https://github.com/elastic/elasticsearch/pull/102426/commits/cc62b3b582557ab7c92c09c72147cee5362f5093) .